tools/ci_build/github/linux/docker/Dockerfile.ubuntu_tensorrt_bin

```diff
@@ -0,0 +1,65 @@
+# --------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------
+# Dockerfile to run ONNXRuntime with TensorRT installed from provided binaries
+
+FROM nvidia/cuda:11.7.1-cudnn8-devel-ubuntu20.04
+
+
+# ONNX Runtime Variables
+ARG ONNXRUNTIME_REPO=https://github.com/Microsoft/onnxruntime
+ARG ONNXRUNTIME_BRANCH=main
+ARG CMAKE_CUDA_ARCHITECTURES=37;50;52;60;61;70;75;80
+
+# Version numbers used to build the name of the tar file containing TensorRT binaries.
+# See: https://docs.nvidia.com/deeplearning/tensorrt/install-guide/index.html#installing-tar
+ARG TAR_TRT_VERSION=8.5.0.9
```

Comment:
Just wondering is it possible to make those versions configurable? Otherwise, we need to update this dockerfile every time we have a new tar file.

Reply:
This is configurable. This is just the default value.

Reply:
I could remove the default values to force users to always set a value. That would probably be ok.
``` dockerfile
# Must set version numbers used to build the tar.gz file name
ARG TAR_TRT_VERSION
ARG TAR_CUDA_VERSION
ARG TAR_CUDNN_VERSION
```

Reply:
yeah, that seems like safest approach. thanks.

Reply:
yeah, make users or build_image.py provide the values is more flexible.  So that we don't need to modify this docker file every time we have a new trt tar file to test.

Reply:
You can modify in this PR or modify it later in other refactor PR? How do you think? 
I think this PR is almost good to go.

Reply:
The build_image.py script is always providing the values, so we wouldn't have to modify the docker file. However, it would be good to force other users of this docker file (those not using our scripts) to enter this information. I'll remove the default values.